packages/astro/src/@types/astro.ts

```diff
@@ -822,6 +822,39 @@ export interface AstroUserConfig {
 		 * ```
 		 */
 		inlineStylesheets?: 'always' | 'auto' | 'never';
+
+		/**
+		 * @docs
+		 * @name build.ssrMode
+		 * @type {boolean}
+		 * @default {false}
+		 * @description
+		 * Defines how the SSR code should be bundled. SSR code for "server"
+		 * will be built in one single file.
```

Comment:
The docs here needs to be updated.

Reply:
Yes! I'm reviewing the docs PR and it mentions `build.split` so I'm assuming that's the correct name for this?